### ISSUE TITLE
error stack trace does not have links in IW if code has been run from the input box

### ISSUE THREAD

**Issue by amunger** [microsoft/vscode-jupyter#10631]
As soon as code is executed from the input box, errors will stop rendering links on the line numbers.

Code that is executed through the input box does not go through the `CodeGenerator`, and does not increase the execution count, so we end up with a mismatch and the `TracebackFormatter` cannot find the code to reference

![image](https://user-images.githubusercontent.com/2019016/176506331-22862859-2ae0-40f8-9f8b-7b51b32ccc2b.png)

**Comment by amunger** [comment]
verification steps:
1. open a python file and execute a `# %%` cell to create an interactive window
2. execute some code from the input box
3. execute these cells from the python file 
 
```
# %%
def raiser():
  raise Exception("error")
# %%
raiser()
```

4. check the resulting error from that cell - the line numbers should be links that will navigate to the correct location in the python file

**Comment by rzhao271** [comment]
The issue doesn't seem fixed using v2022.7.1001841019.
Maybe the fix isn't out yet?

![Screenshot showing the original problem still occurs](https://user-images.githubusercontent.com/7199958/176976354-9729af31-0bf3-4dc7-a3fd-984cef8c1fa6.PNG)

**Comment by rzhao271** [comment]
Verified with today's Insiders
